### ISSUE TITLE
Lightning symbol

### ISSUE THREAD

**Issue by infusion** [KaTeX/KaTeX#1846]
Hello,

people tend to use the lightning symbol ↯  (LaTex: \unicode{x21af}) for contradictions. I switched over from Mathjax, where the latter was possible. Here in KaTeX I tried to use \text{↯}, which works fine but I get the following warning on the console:

```
katex.min.js:1 LaTeX-incompatible input and strict mode is set to 'warn': Unrecognized Unicode character "↯" (8623) [unknownSymbol]
No character metrics for '↯' in style 'Main-Regular'
```

Is there something we can do about it?

Thanks Robert

**Comment by edemaine** [comment]
In LaTeX, this seems to be `\lightning` provided by `stmaryrd`, `MnSymbol`, `fdsymbol`, or `wasysym` packages (all slightly different).  To properly add it to KaTeX, we would need to add it to [the font](https://github.com/KaTeX/katex-fonts).  I don't have a great sense of how easy this is.

You can turn off the warning by adjusting the `strict` setting.

**Comment by infusion** [comment]
Yep, in Tex I tend to use `\lightning` as well. The question is if we need to increase the file size of the fonts for symbols that are rarely used or could the official recommendation be to use `\text` with unicodes. If so, I wonder how to avoid those warnings. Even if the lightning symbol is displayed correctly, I don't know if it conflicts internally somehow to result in that warning. 

In the Mathjax world there exists the possibility to add custom commands which would allow users to personalize their Tex commands, like adding `\lightning` themselves or with addon-packages.

**Comment by kevinbarabash** [comment]
There's a few other symbols that have been requested from `wasysym` so my (weak) preference would be to use that package.  In terms of where to put rarely used glyphs, I think create a new .TTF might make sense.  @infusion which font are you using in the browser to make the symbol show up in `\text`?

**Comment by infusion** [comment]
@kevinbarabash I embed KaTex into my site where I use a Google Font. But I think that does not matter. The important thing is that unicode is used on the site. I tested KaTex with _html entities_ which fails with a parse error. But using `\text` simply adds

```
<span class="mord text"><span class="mord">↯</span></span>
```
to the DOM - which is all fine. I also have no problem using `\text` or add a macro for `\lightning`. My concerns are regarding the warnings. Are they save to ignore or it just luck that it works with the tex snippet I parsed?

**Comment by edemaine** [comment]
The warning means that the symbol is not in the KaTeX font, so it is falling back to your document font (I believe), which can look inconsistent. KaTeX also doesn't know the size of the character, which can lead to incorrect vertical spacing in particular. If neither of these are issues, you can ignore the warnings, or remove them via the `strict` setting.

**Comment by edemaine** [comment]
@497e0bdf29873 That's a separate issue, which is the apparent lack of support for `‖`. If you render `‖` next to `\|` you'll see that KaTeX processes them differently: 
![image](https://github.com/KaTeX/KaTeX/assets/2218736/e86433bf-2813-4c73-84bd-4850446c1988)
